TTHAnalysis/python/plotter/susy-ewkino/2lss/includes/mca_convs.txt

```diff
@@ -0,0 +1,7 @@
+convs  : ZGTo2LG  : xsec : ((LepGood1_mcMatchId==0 && LepGood1_mcPromptGamma==1) || (LepGood2_mcMatchId==0 && LepGood2_mcPromptGamma==1)) ; FillColor=ROOT.kOrange-3, NormSystematic=0.50
+convs  : TGJets   : xsec : ((LepGood1_mcMatchId==0 && LepGood1_mcPromptGamma==1) || (LepGood2_mcMatchId==0 && LepGood2_mcPromptGamma==1)) ; FillColor=ROOT.kOrange-3, NormSystematic=0.50
```

Comment:
maybe i'm wrong, but don't you wanna use the internal conversions from TG (like from WG) too? 
I'm frankly a bit surprised about ZG and TTG, since they should give 3L and only WG/TG have been used in the past for 2lss. Can you check how big are the yields for ZG and TTG?